plugins/org.locationtech.udig.catalog.ui/plugin.properties

```diff
@@ -42,7 +42,7 @@ description = Create a new in-memory service
 import.action.name = Catalog Import Drag and Drop
 
 layerExport.desc = Export all the features in a layer/resource.  The new layer is added to the catalog for later use.
-layerExport.name = Resource to Shapefile
+layerExport.name = All Features to Shapefile
```

Comment:
If we rename this we should care about the user-docs as well (see markdowns in docs folder).

For example have a look at the the current generated docs (using Python and Sphinx):
* https://udig.github.io/docs/user/reference/Resource%20to%20Shapefile%20Wizard.html
* https://udig.github.io/docs/user/getting_started/walkthrough2/ExportingtoShapefile.html

Reply:
I updated the documentation in another commit!